stable/prometheus/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: prometheus
-version: 11.2.3
+version: 11.2.4
```

Comment:
Let's advance this PR as major version update 11.3.0.

Reply:
Okay, done.